src/data/repository/campaign/core/CampaignLogOutcomeRepository.ts

```diff
@@ -6,78 +6,78 @@ export class CampaignLogOutcomeRepository implements OutcomeRepository {
   private outcomes: Outcome[] = [
     {
       id: "a-couriers-job",
-      name: "A courier's job",
       effect: "",
+      translationKey: "outcome.a-couriers-job",
     },
     {
       id: "adamant",
-      name: "Adamant",
       effect: "",
+      translationKey: "outcome.adamant",
     },
     {
       id: "a-hideous-act",
-      name: "A hideous act",
       effect: "",
+      translationKey: "outcome.a-hideous-act",
     },
     {
       id: "a-lovers-request",
-      name: "A lover's request",
       effect: "",
+      translationKey: "outcome.a-lovers-request",
     },
     {
       id: "corrupted-hero",
-      name: "Corrupted hero",
       effect: "",
+      translationKey: "outcome.corrupted-hero",
     },
     {
       id: "curse-breaker",
-      name: "Curse breaker",
       effect: "",
+      translationKey: "outcome.curse-breaker",
     },
     {
       id: "dynamic-duo",
-      name: "Dynamic duo",
       effect: "",
+      translationKey: "outcome.dynamic-duo",
     },
     {
       id: "gentle-dismissal",
-      name: "Gentle dismissal",
       effect: "",
+      translationKey: "outcome.gentle-dismissal",
     },
     {
       id: "guardians-curse",
-      name: "Guardian's curse",
       effect: "As long as you have Guardian's Curse, you cannot unequip the Cursed Bracelet.",
```

Comment:
How can the effects be translated?

Reply:
Both aura and outcome can have effects

Reply:
I did only work on the outcomes so far but already did some effects, st least for those who already have effect-descriptions. nut sure if i will already add texts for every single outcome even though they dont have any effects.

Reply:
Just out curiosity. What speak against to make it like the keywords? For example: 

```
"aura": {
    "core": [
      {
       "id": "ABC",
       "name:": "ABC",
       "effect": "Blabla"
       },
        .....
     ],
    "awakening": [
       {
       "id": "DEF",
       "name:": "DEF",
       "effect": "...."
       },
        .....
     ],
}
```

Reply:
You are totally right. Missed that. @Vortilion if you look at the keyword repository you can see it loads the correct translated data when needed. This allows you to copy the entire object with data to the translations file without implementing specific translations throughout the app itself. Makes it easier to add or adjust text as well. Sorry to say that your current implementation can be done a lot easier if done like that. My bad for missing that. My apologies

Reply:
I dont understand. You really want to refactor every single outcome or aura translation into its own object? I can do that, but that requires a lot of refactoring and boilerplate code, cause only a few outcomes or auras have texts for effects...

Reply:
How about 
{
      id: "bitten",
      effect:
        "Whenever you take a Recall Action, you suffer STUN. This STUN pierces any kind of immunity you have, since it represents your difficulty in handling your actions after sustaining such a serious injury and not a mental disorder due to physical trauma.",
      translationKeys: {
        name: "outcome.bitten",
        effect: "outcome.bitten-effect",
      },
    },

Reply:
Guess I will have to throw away my local changes then.

Reply:
How do you plant to do this now? I can offer my support to do all the stuff in the *.json files. Just tell me how the format should look know. 

Apart from that, just note that the expansions "awakenings" and "apocalypse" have two types of logs (Campaign Log and Story Record). For Story Record there are Outcomes, Status and Unfoldings. And for the Campaign Log there are Outcomes, Status and Auras.

Reply:
Done in new commits.